README.md

```diff
@@ -37,10 +37,15 @@ matter what URL the browser requests.
 
 ## Env Variables
 
-- `VITE_SIGNING_SERVICE_URL`: Optional variable to point to a specific signing backend service URL. If undefined, it
+- `VITE_SIGNING_SERVICE_PATH`: Optional variable to point to a specific signing backend service URL. If undefined, it
```

Comment:
I assume you decided to rename this variable as the values for `VITE_SIGNING_SERVICE_PATH` will not always be URLs but can also be just paths?

Reply:
Yes, I forgot to describe this in the PR description. Yes, I defined new variables to have a smooth transition to the new system. I already defined them in Netlify.

Reply:
The definition for production is `/api/production` and for staging and branch deploys is `/api/staging`.